### ISSUE TITLE
How to output unwrap coordinates?

### ISSUE THREAD

**Issue by lujunfeng1** [brucefan1983/GPUMD#663]
When running gpumd, I want to output the unwrap coordinates. Here's how to set it up.
First, change the atom.position_per_atom replace with atom.unwrapped_position in measure.cu.
Second, recompile the software with the make command in the src directory.
Now , you can get the coordinates of the unwrap with the command dump_position.
![dump_position](https://github.com/brucefan1983/GPUMD/assets/87063890/df72952a-21c5-441f-9d60-3c049be043ec)

**Comment by brucefan1983** [comment]
Yes, this is a simple fix. we can improve the relevant keywords in the future to make them more flexible.

**Comment by brucefan1983** [comment]
solved in #976
